### PR TITLE
Feature/10 add payment to orderinfo

### PR DESCRIPTION
**Pull request by bennsen**

### CODE REVIEW

File: src/Nodes/Payment/Card.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
```
Comment (juanitolaguna) [inline src/Nodes/Payment/Card.php:3]: strict_types not used in this library

File: tests/Nodes/Payment/PaymentTermTest.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
```
Comment (juanitolaguna) [inline tests/Nodes/Payment/PaymentTermTest.php:3]:
```suggestion
```

File: tests/Nodes/Payment/PaymentTest.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
```
Comment (juanitolaguna) [inline tests/Nodes/Payment/PaymentTest.php:3]:
```suggestion
```

File: src/Nodes/Payment/Card.php (hunk 1 above)
Comment (juanitolaguna) [reply src/Nodes/Payment/Card.php:3]:
```suggestion
```

**Review by juanitolaguna — CHANGES_REQUESTED** [review]
Remove strict_types for consistency.

File: src/Nodes/Payment/Card.php (hunk 1 above)
Comment (Naugrimm) [reply src/Nodes/Payment/Card.php:3]: As these are new files, I do not see any problems in declaring strict types. 

@juanitolaguna do you see any issues besides consistency?

**Comment by Naugrimm** [comment]
@bennsen The tests do not run completely. There are two errors regarding the BankCode/BIC

File: src/Nodes/Payment/Card.php (hunk 1 above)
Comment (juanitolaguna) [reply src/Nodes/Payment/Card.php:3]: @Naugrimm no issues, if you are ok with that.

**Comment by bennsen** [comment]
> @bennsen The tests do not run completely. There are two errors regarding the BankCode/BIC

I missed to commit the BankCode changes... now it should work. please have a look

**Pull request merged**